scripts/setupCards.py

```diff
@@ -131,6 +151,21 @@
     cb.cp().channel([chn]).backgrounds().ExtractShapes(inputfile, '$BIN/$PROCESS', '$BIN/$PROCESS_$SYSTEMATIC') 
     cb.cp().channel([chn]).signals().ExtractShapes(inputfile, '$BIN/$PROCESS', '$BIN/$PROCESS_$SYSTEMATIC') 
 
+
+
+for expr, replacement in replacement_dict.items():
```

Comment:
Does this actually produce the datacards that you expect? I would not expect it to add the systematics for the renamed processes, since you already put the names after the renaming that you're doing here in the systematics script. So when you call AddSystematics, it's not going to add the relevant uncertainties for the processes that don't exist at that time (ie with the names that you're going to set here). Therefore I would expect the uncertainties to be missing for most of the H processes. I guess it's not really a comment for this part of the code, which looks fine, but for the process names in the systematics file

Reply:
Fixed in the subsequent commit